yang/frr-staticd.yang

```diff
@@ -39,6 +39,49 @@ module frr-staticd {
       "'Static' routing pseudo-protocol.";
   }
 
+  typedef optional-src-prefix {
+    type union {
+      type inet:ipv6-prefix;
+      type string {
+        pattern
+          '';
+      }
+    }
+  }
+
+  grouping staticd-prefix-attributes {
+
+    list path-list {
+      key "distance tag table-id";
+
+      leaf distance {
+        type frr-rt:administrative-distance;
+        description
+          "Admin distance associated with this route.";
+      }
+      leaf tag {
+        type uint32 {
+          range "0..4294967295";
```

Comment:
this is already the max range of a uint32

Reply:
Actually at present from the CLI staticd has below range check around ip_route_cmd
tag (1-4294967295)
table (1-4294967295)
So i have used the same in yang also.
If still required, then we will update it.

Reply:
Yes, that's because the CLI requires a range for all numeric inputs, and the desired range is that of a uint32_t. The fact that you're using a uint32_t at all means this limitation is implicit. So, please remove the range statement here.

Reply:
removed